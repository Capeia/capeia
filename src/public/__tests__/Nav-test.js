// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<Nav>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('../Nav')
    const Component = require('../Nav').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
