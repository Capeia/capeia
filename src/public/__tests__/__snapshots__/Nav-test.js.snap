// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav> renders correctly 1`] = `
<nav
  className="main-nav"
  role="navigation"
>
  <ul>
    <li>
      <a
        activeClassName="active"
        href="/stem-cells"
      >
        <img
          alt=""
          src="sections/stem_cells.svg"
        />
        Stem Cells
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/brain-and-nerves"
      >
        <img
          alt=""
          src="sections/brain_nerves.svg"
        />
        Brain & Nerves
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/cancer"
      >
        <img
          alt=""
          src="sections/cancer.svg"
        />
        Cancer
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/zoology"
      >
        <img
          alt=""
          src="sections/zoology.svg"
        />
        Zoology
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/paleobiology"
      >
        <img
          alt=""
          src="sections/paleontology.svg"
        />
        Paleobiology
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/planetary-science"
      >
        <img
          alt=""
          src="sections/planetary_science.svg"
        />
        Planetary Science
      </a>
    </li>
  </ul>
</nav>
`;
